core/src/test/kotlin/integration_tests/LoadTestsFromExternalisedFiles.kt

```diff
@@ -227,7 +227,7 @@ class LoadTestsFromExternalisedFiles {
 
         assertThat(results.successCount).isEqualTo(1)
         assertThat(results.success()).withFailMessage(results.report()).isTrue()
-        System.clearProperty(LOCAL_TESTS_DIRECTORY)
+        System.clearProperty(EXAMPLE_DIRECTORIES)
```

Comment:
This should be in a finally block, both here and anywhere else this property is set. That way if the test fails, the property will get unset and will not impact any other tests.

Reply:
Thanks, added logic to clear system properties in test files.